src/client/camera.h

```diff
@@ -133,13 +142,18 @@ class Camera
 	void drawWieldedTool(irr::core::matrix4* translation=NULL);
 
 	// Toggle the current camera mode
-	void toggleCameraMode() {
-		if (m_camera_mode == CAMERA_MODE_FIRST)
-			m_camera_mode = CAMERA_MODE_THIRD;
-		else if (m_camera_mode == CAMERA_MODE_THIRD)
-			m_camera_mode = CAMERA_MODE_THIRD_FRONT;
-		else
-			m_camera_mode = CAMERA_MODE_FIRST;
+	void toggleCameraMode();
+
+	// Set camera mode restrictions
+	inline void setCameraModeRestrictions(const CameraModeRestrictions restrictions)
```

Comment:
okay it's const, but where is the ref now ? :)

Reply:
Done.